lib/aggregate.rb

```diff
@@ -23,4 +23,40 @@
 require "aggregate/container"
 
 module Aggregate
+  class << self
+    attr_writer :configuration
```

Comment:
I was discussing this with Chad - I think we it's fine. Chad mentioned that Aggregate isn't directly called anywhere and this is a class to the module Aggregate, so it should be safe.

Separately, this seemed like a pretty common design pattern / naming convention when I first looked into gem configurations

https://robots.thoughtbot.com/mygem-configure-block
https://github.com/carrierwaveuploader/carrierwave#configuring-carrierwave
Main guide I used: http://brandonhilkert.com/blog/ruby-gem-configuration-patterns/